.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-16.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get install libstdc++6
+          sudo apt-get install libhttp-parser-dev
+          sudo apt-get install libssh2-1-dev
```

Comment:
These three could be done in one command, which is probably faster.
```
sudo apt-get install -y libstdc++6 libhttp-parser-dev libssh2-1-dev
```

(The `-y` flag should be passed to answer yes to any otherwise-interactive prompts.)

Reply:
Okay done.